Dockerfile-web

```diff
@@ -0,0 +1,18 @@
+FROM tiangolo/node-frontend:10 as build-stage
```

Comment:
Was this file the attempt to have your frontend code run on an nginx server in a Docker container on its own? For now we can assume your frontend code will be uploaded to some hosting solution and be like a static website that will eventually make requests to a cloud-hosted backend

Reply:
If you would like to see a resource of running a Node.js server in a Docker container, this one was good start that I pored through: https://nodejs.org/en/docs/guides/nodejs-docker-webapp/